drivers/bridge/bridge.go

```diff
@@ -959,6 +961,9 @@ func (d *driver) CreateEndpoint(nid, eid string, ifInfo driverapi.InterfaceInfo,
 	if endpoint.addrv6 == nil && config.EnableIPv6 {
 		var ip6 net.IP
 		network := n.bridge.bridgeIPv6
+		if config.AddressIPv6 != nil {
+			network = config.AddressIPv6
+		}
 		ones, _ := network.Mask.Size()
 		if ones <= 80 {
 			ip6 = make(net.IP, len(network.IP))
```

Comment:
`ip6` is set only for the condition of `ones <= 80`. it is possible that it can be nil in cases when the config.AddressIPv6 doesnt fall under the condition. Should this be checked ?